packages/dmn-editor/src/diagram/nodes/Nodes.tsx

```diff
@@ -239,7 +239,7 @@ export const InputDataNode = React.memo(
           onDoubleClick={triggerEditing}
           onKeyDown={triggerEditingIfEnter}
           style={alternativeSvgStyle}
-          className={`kie-dmn-editor--input-data-node ${selectedAlternativeClass}`}
+          className={`kie-dmn-editor--input-data-node ${additionalClasses} ${selectedAlternativeClass}`}
```

Comment:
With this change we have code:
```
        <div
          onDoubleClick={triggerEditing}
          onKeyDown={triggerEditingIfEnter}
          style={alternativeSvgStyle}
          className={`kie-dmn-editor--input-data-node ${additionalClasses} ${selectedAlternativeClass}`}
          ref={ref}
          tabIndex={-1}
          data-nodehref={id}
          data-nodelabel={inputData["@_name"]}
        >
          {/* {`render count: ${renderCount.current}`}
          <br /> */}
          <div className={`kie-dmn-editor--node ${additionalClasses}`}>
            <InfoNodePanel isVisible={!isTargeted && shouldActLikeHovered} />
            ....
```

I know we have currently twice `${additionalClasses}` used, I thought it could be enough to keep `${additionalClasses}` only in the top level `div`, however in such scenario, node is disabled, user can not add connection to it, however the text is not 'dimmed', see the screenshot below.

![Screenshot 2024-03-19 124932](https://github.com/ljmotta/kie-tools/assets/8044780/368bca2b-8496-414b-8cd4-c2ba7a0106d2)